### PR TITLE
fix: bug while adding in edit/view modal

### PR DESCRIPTION
**Pull request by kartik162000**
fix bug while adding member, in edit/view modal

### CODE REVIEW

File: src/components/elements/AccountSettingsModal.js [hunk 1]
```diff
@@ -273,6 +260,13 @@ function AccountSettingsModal({ open, setOpenSettings }) {
         },
       })
       .then(() => {
+        setProjectInfo({
```
Comment (sugumar-mani) [inline src/components/elements/AccountSettingsModal.js:263]: you need to refetch the collaborators list. We should always represent what is there in the BE. 
What if BE returned 200 and did not delete the user due to some issue?

Comment (kartik162000) [reply src/components/elements/AccountSettingsModal.js]: this is resolved. can u please check in latest commit

**Comment by bharathraj25** [comment]
LGTM

**Review by bharathraj25 — APPROVED** [review]
LGTM

**Pull request merged**